lib/v2/dgjyw/index.js

```diff
@@ -0,0 +1,67 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const params = ctx.path;
+
+    const rootUrl = 'http://www.dgjyw.com';
```

Comment:
```suggestion
    const rootUrl = 'https://www.dgjyw.com';
```
HTTP traffics will be redirected to HTTPS